packages/now-routing-utils/src/index.ts

```diff
@@ -150,13 +148,7 @@ export function getTransformedRoutes({
     if (typeof headers !== 'undefined') {
       routes.push(...convertHeaders(headers));
     }
-    if (typeof cleanUrlsRewrites !== 'undefined') {
-      routes.push(...cleanUrlsRewrites);
-    }
-    if (
-      typeof cleanUrlsRewrites !== 'undefined' ||
-      typeof rewrites !== 'undefined'
-    ) {
+    if (typeof rewrites !== 'undefined') {
```

Comment:
This same statement exists again below in line 154. Let's combine them.

Reply:
Done 👍